app/src/test/java/sw19/moring03/paint/FillTest.java

```diff
@@ -0,0 +1,133 @@
+package sw19.moring03.paint;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import sw19.moring03.paint.utils.PointF;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import sw19.moring03.paint.tools.FillTool;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FillTest {
+    private Canvas canvas;
+    private Paint paint;
+
+    @Before
+    public void startUp() {
+        canvas = new Canvas();
+        paint = new Paint();
+    }
+
+    @Test
+    public void testSimpleFillTool() {
+        FillTool tool = new FillTool();
+        tool.addPoint(new PointF(15, 15));
+        tool.addPoint(new PointF(20, 20));
```

Comment:
pls also check if the size of the point list is correct (1 not 2)

Reply:
I found that the only thing that makes sense in this test is to check the size of the point list.
Drawing makes no sense here since we don't provide the `FillTool` with a bitmap. So it cannot calculate the necessary fills and draw anything.

Fixed with commit 7225edaa72.